ftw/upgrade/command/jsonapi.py

```diff
@@ -100,8 +100,9 @@ def func_wrapper(args):
         auth_value = args.auth or default_auth
         if auth_value:
             if len(auth_value.split(':')) != 2:
-                print 'ERROR: Invalid authentication information "{0}".'.format(
-                    auth_value)
+                print ('ERROR: Invalid authentication'
```

Comment:
@jone, does this make the code more readable? According to PEP8 it's ok to ignore a particular guideline if it makes the code less readable (the guideline in question is the maximum line length). What do you think?